docs/source/aws/examples/asgnotused.rst

```diff
@@ -0,0 +1,59 @@
+AMI - ASG Garbage Collector
+====================================
+ASG garbage collector which mean that:
+
+Check if an ASG has minSize = 0 and DesiredCapacity = 0
+Mark the ASG as ops to delete.
```

Comment:
just need to change the description that its a notify now, not a delete

Reply:
sure! changed it